deleting-clusters-and-cleaning-up.md

```diff
@@ -0,0 +1,39 @@
+# Deleting clusters and cleaning up
+
+## Deleting clusters
+Clusters managed by Cluster API can be deleted in the same way as any other Kubernetes object.
+
+To delete an individual cluster by name run:
+
+```bash
+kubectl delete cluster docker-cluster-one
+```
+This delete operation ensures that the cluster and all of the objects that form part of the cluster e.g. Machines, Cluster Infrastructure, are correctly deleted. 
+
+To delete all clusters managed by Cluster API run:
+
+```bash
+kubectl delete clusters --all -A
+```
+
+## Cleaning up resources created by this tutorial
+The management cluster in this tutorial is not managed by Cluster API until [you make it self-hosted](./docker-cluster-self-hosted.md).
+
+To clean up the management cluster before this step run:
```

Comment:
I think I know what you mean, but this sounds confusing

Reply:
The self-hosted thing also moves the management stuff back at the end

So I wonder if we should just simplify this to "this is how you just get rid of evertyhing"

Reply:
I could seperate this out into different docs - but the idea here is that these are independent sections, with one for deleting clusters and another for cleaning everything up e.g. if they get into a bad state and want to restart the right approach is `kind delete clusters --all`

Reply:
Yup agree. I'm totally fine with different sections for regular happy-path delete cluster vs. cleanup.

I just wonder if our cleanup should simply by `kind delete clusters --all`. I'm not sure it's necessary to potentially have a step where we first delete the management cluster with kind and then all others.